### ISSUE TITLE
your take on falcor and graphql ?

### ISSUE THREAD

**Issue by niccolox** [devinus/poison#116]
hey there

am wondering about your thinking on how Poison fits with Graphql and the Elixir implementation absinthe and/or Falcor, which doesnt seem to have an Elixir implementation

I see absinthe doesnt have poison as a dependency (but I guess its often used with it)

looking at doing something with Maxwell https://github.com/zhongwencool/maxwell and Poison and it feels like I am trying to solve the same problem graphql(Facebook)/ json graph(Netflix/falcor) solve

basically querying and decoding json api

thanks for your consideration in this matter

**Comment by devinus** [comment]
They're really cool technologies, but Poison is just a JSON parser. Use GraphQL if you're building an API.
